packages/flutter_tools/.analysis_options

```diff
@@ -0,0 +1,18 @@
+# A temporary file to do some validation in the flutter_tools project. This will
+# go away once we unify `flutter analyze` and analysis server based products.
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  errors:
+    todo: ignore
+linter:
+  rules:
+    - avoid_empty_else
+    - always_declare_return_types
+    - always_specify_types
+    - annotate_overrides
+    - sort_unnamed_constructors_first
+    - unnecessary_brace_in_string_interp
+    - unnecessary_getters_setters
```

Comment:
I didn't want another copy of that file to maintain :) I just need the few that I hit on a regular basis (and see failures on travis with).